src/System.Text.Primitives/System/Text/FormattingData.cs

```diff
@@ -142,6 +503,9 @@ public enum Symbol : ushort
             PlusSign = 14,          
             NaN = 15,
             Exponent = 16,
+            ExponentSecondary = 17,
+            Invalid = 18, // invalid character for parsing
```

Comment:
Can we avoid adding these two here? They are not symbols. It makes the public API confusing. Especially for formatting where symbol values are passed in (and passing in Invalid would be invalid).